### PR TITLE
AR Core 1.16 update

### PR DESCRIPTION
**Pull request by moljac**

### CODE REVIEW

**Comment by jparismorgan** [comment]
Thanks @moljac! Are you planning to add SceneForm 1.16 as well?

**Comment by moljac** [comment]
@julianparismorgan 
I would if it would be available:

https://maven.google.com/web/index.html

The latest version is 1.15. Checked few days ago and now 8-)

File: Android/ARCore/source/Google.ARCore.csproj [hunk 1]
```diff
@@ -2,8 +2,7 @@
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <AndroidCodegenTarget>XAJavaInterop1</AndroidCodegenTarget>
-    <TargetFrameworkVersion>v9.0</TargetFrameworkVersion>
-    <ReleaseVersion>1.14.0</ReleaseVersion>
```
Comment (JonDouglas) [inline Android/ARCore/source/Google.ARCore.csproj:6]: Should this `<ReleaseVersion>` be removed or bumped?

**Review by JonDouglas — APPROVED** [review]
Added a comment regarding the version being removed, but otherwise LGTM

File: Android/ARCore/source/Google.ARCore.csproj (hunk 1 above)
Comment (moljac) [reply Android/ARCore/source/Google.ARCore.csproj:6]: @JonDouglas Seems that cake script or IDE removes it. This is 2nd time that happened and I did not do it on purpose (manually).

Comment (moljac) [reply Android/ARCore/source/Google.ARCore.csproj:6]: restored

**Comment by WouterStraver** [comment]
@moljac Currently Sceneform 1.17 is available at  https://mvnrepository.com/artifact/com.google.ar.sceneform . Are you planning to implement this?

**Comment by WouterStraver** [comment]
@moljac would it be possible to update scene form? This would help me a lot!

**Comment by WouterStraver** [comment]
@moljac Any update plans for Sceneform 1.17 so far? This would help me a lot.

**Pull request merged**